quinn-proto/src/tests/util.rs

```diff
@@ -144,21 +144,19 @@ impl Pair {
                 );
                 continue;
             }
-            if x.contents[0] & packet::LONG_HEADER_FORM == 0 {
-                let spin = x.contents[0] & packet::SPIN_BIT != 0;
+            if x.1[0] & packet::LONG_HEADER_FORM == 0 {
```

Comment:
IMO we should unpack the tuple in the loop header here.

Reply:
done